BraveRewardsTests/DateExtensionsTests.swift

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+@testable import BraveRewardsUI
+
+class DateExtensionsTests: XCTestCase {
+
+  func testCurrentMonth() {
+    let february = dateFrom(string: "2018-02-22")
+    // Make sure tests are running on default english locale
+    XCTAssertEqual(february.currentMonthName, "February")
+  }
+  
```

Comment:
Based on that previous comment about polish months: Is it possible add a locale specific test here too? :)  Not a problem if not, just as long as we test it manually

Reply:
done

Reply:
done